content/millennium/r4/encounters/encounter.md

```diff
@@ -97,3 +97,56 @@ The common [errors] and [OperationOutcomes] may be returned.
 [`_count`]: http://hl7.org/fhir/R4/search.html#count
 [errors]: ../../#client-errors
 [OperationOutcomes]: ../../#operation-outcomes
+
+## Create
+
+Create an individual Encounter.
+
+    POST /Encounter
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', 'Content-Type': 'application/json+fhir'} %>
+
+### Body Fields
+
+<%= definition_table(:encounter, :create, :r4) %>
+
+### Example
+
+#### Request
+
+    POST https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Encounter
+
+#### Body
+
+  <%= json(:r4_encounter_create) %>
+
+#### Response
+
+<%= headers status: 201 %>
+<pre class="terminal">
+    Cache-Control → no-cache
```

Comment:
let's use colons instead of weird arrow characters

Reply:
changed: https://github.com/cerner/fhir.cerner.com/pull/239/commits/9c48980f8f6217f0ba08e6a136317e557903f83e

Reply:
can we remove the leading spaces too?

Reply:
removed: https://github.com/cerner/fhir.cerner.com/pull/239/commits/151548d450d95be176f9e63386de7cbd34a6e28a